src/parquet/encodings/plain-encoding.h

```diff
@@ -71,6 +71,22 @@ inline int PlainDecoder<parquet::Type::BYTE_ARRAY>::Decode(ByteArray* buffer,
   return max_values;
 }
 
+// Template specialization for FIXED_LEN_BYTE_ARRAY
+template <>
+inline int PlainDecoder<parquet::Type::FIXED_LEN_BYTE_ARRAY>::Decode(FixedLenByteArray* buffer,
```

Comment:
I will add that now. Actually its rare for these types to be dictionary encoded.
I tested this on my side and it seems to work okay. We should definitely add tests after PARQUET-435